src/gs-page.c

```diff
@@ -581,6 +588,7 @@ gs_page_update_app (GsPage *page, GsApp *app, GCancellable *cancellable)
 	helper->app = g_object_ref (app);
 	helper->page = g_object_ref (page);
 	helper->cancellable = g_object_ref (cancellable);
+	helper->settings = g_object_ref (g_settings_new ("org.gnome.software"));
```

Comment:
The `g_settings_new` call already gives you a new object. If you call `g_object_ref` here, they you will be leaking memory.

Reply:
Ah, I looked for a `utils` file but couldn't find it. I realized it I was looking in `src/` only but there is one in `lib/`